e2e_playwright/lazy_loaded_modules.py

```diff
@@ -23,24 +24,35 @@
     "pydeck",
     "altair",
     "graphviz",
-    "watchdog",
     "matplotlib",
     "pandas",
     "pyarrow",
+    "numpy",
```

Comment:
super nit: we should start to add these things in alphabetically so that it is easier for humans to scan the list very easily 😁

Reply:
Sorted 👍